src/index.html

```diff
@@ -7,8 +7,33 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,700">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="navLink" href="#">
+        <img class="logo"
+        src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSFj1y1pQkmogzxonVdGSNhFu8RLNu4Pmwxhekuf7jeAWoQAdsnTlYSYGGfiqPDspBHKEY&usqp=CAU"
+        alt=" ">
+      </a>
+      <nav class="navMenu">
+        <a class="navLink isActive" rel="stylesheet"
+      href="https://www.google.com/">Apple</a>
```

Comment:
it should be as a list. so use next structure of tags
```
<ul>
  <li>
    <a>
      text
    </>
  </li>
</ul>
```

also use BEM recommendations for naming classes
nav, nav__list, nav__item, nav__link